lib/include/kickcat/Mailbox.h

```diff
@@ -186,7 +187,9 @@ namespace kickcat::mailbox::response
         void enableCoE(CoE::Dictionary&& dictionary);
         CoE::Dictionary& getDictionary(){return dictionary_;}
 
-        std::tuple<SyncManagerConfig, SyncManagerConfig> configureSm();
+        hresult configureSm();
+        bool is_sm_config_ok();
+        void set_sm_activate(bool is_activated);
```

Comment:
I think we can hide the internal details: set_sm_activate -> activate, is_sm_config_ok() -> is_config_ok(), configureSm() -> configure(), and so on